### ISSUE TITLE
[Feature]: Add the note to current folder

### ISSUE THREAD

**Issue by gerroon** [mgmeyers/obsidian-kanban#865]
### Goal or desired outcome of this feature

THe notes from the cards will be added to the current folder

### Describe the feature

The plugin creates a note from the current card in the current folder as an option in the settings.

### Can you think of any alternatives or work-arounds?

Add the note and manually move the notes to where the kanban note is

**Comment by AmNotAGoose** [comment]
+1, this would be great

**Comment by Dzoukr** [comment]
+1, OR ... even better... let me specify the relative folder (based on current location of active kanban board) 🙏

**Comment by gerroon** [comment]
> You can just set the `Default location for new notes` to `Same folder as the current` in `Settings > Files and links`.

Yes but what if the user does not want that to be a global behavior.

**Comment by HamzaBenyazid** [comment]
+1, I need this. I have multiple kanban boards in my vault, and I can only specify a note global directory.

**Comment by adamsmd** [comment]
There is a PR (#1052) that implements this, but seems to have been closed due to not having a UI to control the option.  (It currently ignores the global option and always puts in the current folder.)

Maybe someone could use that as a base on which to add options in the GUI to control whether it is enabled.

**Comment by thraizz** [comment]
This is in the plugin and can be closed:

![Image](https://github.com/user-attachments/assets/c56c3b08-00f5-49ec-a8e3-2c32d2e69dd9)

**Comment by gerroon** [comment]
See above "Yes but what if the user does not want that to be a global behavior."


> This is in the plugin and can be closed:
> 
> ![Image](https://github.com/user-attachments/assets/c56c3b08-00f5-49ec-a8e3-2c32d2e69dd9)

**Comment by thraizz** [comment]
My bad. I've misread the feature request, sorry.

**Comment by BugCommitter** [comment]
+1, I need this. I hope to be able to set relative paths, such as './Notes'.
